packages/website/src/components/Playground.tsx

```diff
@@ -60,133 +74,151 @@ function Playground(): React.JSX.Element {
     }
   }, []);
 
+  useEffect(() => {
+    if (windowSize === 'mobile') {
+      playgroundMenuRef.current?.collapse();
+    } else if (windowSize === 'desktop') {
+      playgroundMenuRef.current?.expand();
+    }
+  }, [windowSize, playgroundMenuRef]);
```

Comment:
why do you need the ref as a dependency here? doesn't it keep the same reference on each re-render anyway?

```suggestion
  }, [windowSize]);
```